src/index.html

```diff
@@ -1,17 +0,0 @@
-<!DOCTYPE html>
-<html lang="en">
-    <head>
-        <title></title>
-        <meta charset="UTF-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1">
-        <meta name="viewport" content="user-scalable=no, width=device-width,
-      initial-scale=0.5, minimum-scale=0.5, maximum-scale=0.5" />
-        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css" integrity="sha384-rwoIResjU2yc3z8GV/NPeZWAv56rSmLldC3R/AZzGRnGxQQKnKkoFVhFQhNUwEyJ" crossorigin="anonymous">
-        <link href="https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300" rel="stylesheet">
-    <body>
-        <div id="root" class="container"></div>
-        <script src="https://code.jquery.com/jquery-3.1.1.slim.min.js" integrity="sha384-A7FZj7v+d/sdmMqp/nOQwliLvUsJfDHW+k9Omg/a/EheAdgtzNs3hpfag6Ed950n" crossorigin="anonymous"></script>
```

Comment:
It depends on where the library is being hosted and what kind of reference to the library is being made ( latest version always or a specific version )

If the library comes from CDN hosts like google then it will have : 
CDN allows to get the library from a server on the nearest location from where the client is requesting the site.
more availability, 
best caching 
avoid requesting our server for the library.